package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "express-ajv-swagger-validation",
-  "version": "1.2.0",
-  "description": "Input validation using Swagger (Open API) and ajv",
+  "name": "openapi-validator-middleware",
+  "version": "0.0.0",
+  "description": "Fast input validation middleware using OpenAPI definitions file and ajv",
```

Comment:
OpenAPI or Swagger might be more helpful, as many people not aware of Swagger being OpenAPI 2.0

Reply:
changed

Reply:
Common style for the last few years was "OpenAPI (formerly Swagger)" but its been 5 years and at some point we have to accept that a new thing has a new name.

Reply:
Many people still tend to call it swagger file, anyway having both won't hurt. no?

Reply:
And in our case we actually explicitly support both, so saying "OpenAPI 2.0 (formerly Swagger) and 3.0" would be the most accurate way to put it.

Reply:
changed again :)

Reply:
so are we good to go?